src/main/kotlin/car/view/ResultView.kt

```diff
@@ -0,0 +1,35 @@
+package car.view
+
+import car.domain.history.CarMoveHistoryCollection
+import car.domain.CarCollection
+import car.domain.winner.WinnerCollection
+import car.view.printer.CarMoveHistoryPrettyPrinter
+
+class ResultView(
+    private val carCollection: CarCollection,
+    private val input: Input
+) {
+    fun print() {
+        val list: List<CarMoveHistoryCollection> = carCollection
+            .map { it.historyCollection }
+
+        repeat(input.tryCount) { index ->
+            printCarPositionHistory(list, index)
+            println()
+        }
+
+        printWinners(WinnerCollection(carCollection))
+    }
+
+    private fun printCarPositionHistory(carMoveHistoryCollectionList: List<CarMoveHistoryCollection>, index: Int) {
+        for (carMoveHistoryCollection in carMoveHistoryCollectionList) {
+            val pretty = CarMoveHistoryPrettyPrinter(carMoveHistoryCollection[index], input.prettyPositionString)
+            print(pretty)
+            println()
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
            println(pretty)
```